### PR TITLE
Port crm_compassion

### PR DESCRIPTION
**Pull request by ecino**
Don't merge until `sponsorship_compassion` is ported and working, as this module does not work without it.

Port crm_compassion.

### CODE REVIEW

**Comment by ecino** [comment]
You can review this one as well.

File: crm_compassion/model/event_compassion.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+        return {
+            'type': 'event',
+            'event_id': self.id,
+            'analytic_id': analytic_id,
+        }
+
+    def _find_parent_analytic(self):
+        analytics_obj = self.env['account.analytic.account']
+        categ_id = analytics_obj.search(
+            [('name', 'ilike', self.type)])[0].id
```
Comment (steveferry) [inline crm_compassion/model/event_compassion.py]: `search(..., limit=1).id` ?

File: crm_compassion/model/event_compassion.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+            [('name', '=', self.year), ('parent_id', '=', categ_id)]).ids
+        if not acc_ids:
+            # The category for this year does not yet exist
+            acc_ids = [analytics_obj.create({
+                'name': self.year,
+                'type': 'view',
+                'code': 'AA' + self.type[:2].upper() + self.year,
+                'parent_id': categ_id
+            }).id]
+        return acc_ids[0]
```
Comment (steveferry) [inline crm_compassion/model/event_compassion.py]: `acc_ids = [analytics_obj.create({...}).id]` is not really clear. Maybe rename `acc_ids` in `acc_id`, use parameter `limit=1` in the search and remove the brackets `acc_id = analytics_obj.create({...}).id`

**Comment by ecino** [comment]
Pull request corrected.

**Pull request merged**